src/EnergyPlus/WindowManager.hh

```diff
@@ -153,6 +163,43 @@ namespace WindowManager {
 
     //****************************************************************************
 
+    void GetHeatBalanceEqCoefMatrixSimple(EnergyPlusData &state,
+                                          int const ConstrNum,     // Construction number
+                                          int const nglasslayer,   // Number of glass layers
+                                          Array2D<Real64> &Aface,  // Coefficient in equation Aface*thetas = Bface
+                                          Array1D<Real64> &Bface,  // Coefficient in equation Aface*thetas = Bface
+                                          Array1D<Real64> hr,      // Radiative conductance (W/m2-K)
+                                          Array1A<Real64> hgap // Gap gas conductive conductance (W/m2-K)
+    );
+
+    //****************************************************************************
+
+    void GetHeatBalanceEqCoefMatrix(EnergyPlusData &state,
```

Comment:
The code chunk computing the coefficient matrix of the heat balance equations is separated out from the function SolveForWindowTemperatures.